core/contractsdk/java/src/main/java/com/baidu/xuper/Context.java

```diff
@@ -1,18 +1,38 @@
 package com.baidu.xuper;
 
+import java.util.HashMap;
+import java.util.ArrayList;
 import java.util.Map;
 
+import com.baidu.xuper.contractpb.Contract;
+
 public interface Context {
     public Map<String, byte[]> args();
 
     public String caller();
 
+    public ArrayList<String> authRequire();
+
     public void putObject(byte[] key, byte[] value);
 
     public byte[] getObject(byte[] key);
 
     public void deleteObject(byte[] key);
 
+//    public Contract.IteratorResponse newIterator(ByteString start, ByteString limit);
```

Comment:
`ByteString` is an internal type of protobuf, it would be better to use java's native `byte[]` type

Reply:
fixed